recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.23.0" %}
+{% set version = "0.23.X" %}
```

Comment:
shouldn't it be 0.23.1 ?

Reply:
No, I haven't tagged on the branch yet. I wanted to test if these builds succeed before tagging.

Reply:
I wouldn't merge this PR anyway, I'd wait for the bot to create the PR once I draft a release on our repo.

Reply:
Ah ok, thanks. 

> I wanted to test if these builds succeed before tagging.

wise :)